.buildkite/pipeline.yml

```diff
@@ -42,6 +42,9 @@ steps:
         agents:
           provider: "gcp"
 
+      - label: "Detect FIPS crypto imports"
```

Comment:
I see `check-ci` step and I wonder whether this should follow the same naming convention or being added to the above step since it runs `mage check:detectFIPSCryptoImports`? Or maybe rename the script with `check_detect_fips_crypto.sh`?